components/consentRequestForm/__snapshots__/index.test.jsx.snap

```diff
@@ -140,7 +140,7 @@ exports[`Consent Request Form Renders a consent request form with multiple purpo
           placeholder="Consent expiry date"
           readonly=""
           type="text"
-          value="Forever"
+          value="June 23, 2022"
```

Comment:
Careful here, unless you're [mocking time](https://jestjs.io/blog/2020/05/05/jest-26#new-fake-timers). (this is now default in [Jest 27](https://jestjs.io/blog/2021/05/25/jest-27#:~:text=Another%20default%20that%20we%20are%20changing%20affects%20Fake%20Timers%20aka%20Timer%20Mocks))

Reply:
What do you mean? This value is set from the expiration date from the mock consent request.

Reply:
Ah! So it is! I looked at the mockSignedVc request accidentally last night 🤦🏻‍♀️

So if I understand the diff correctly, the date wasn't previously being used so defaulted to Forever, now it is, so it's changed the snapshot to reflect?

Reply:
Aha, okay! Looks good then. My other comments are just ways you could improve the code